pkg/operator/manager.go

```diff
@@ -75,6 +77,14 @@ func RunOperator(cmd *cobra.Command, args []string) {
 	// 	log.Warnf("Failed ExposeMetricsPort: %s", err)
 	// }
 
+	_, ok := os.LookupEnv("ENABLE_NOOBAA_ADMISSION")
+	if ok {
```

Comment:
this will only check if the env exists. you also need to check its value

```suggestion
	enable_addmission, ok := os.LookupEnv("ENABLE_NOOBAA_ADMISSION")
	if ok && enable_admission == "true" {
```